src/Tools/ExternalAccess/FSharp/Navigation/FSharpNavigationOptions.cs

```diff
@@ -1,17 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-#nullable disable
-
-using System;
-using Microsoft.CodeAnalysis.Options;
-
-namespace Microsoft.CodeAnalysis.ExternalAccess.FSharp.Navigation
-{
-    [Obsolete]
-    internal static class FSharpNavigationOptions
```

Comment:
Is this no longer used?

Note that External Access users will not have access to `Option2<T>`.

Reply:
I verified it's no longer used.